app/src/main/java/com/github/polybooks/activities/SaleInformationActivity.kt

```diff
@@ -23,6 +24,83 @@ class SaleInformationActivity: AppCompatActivity() {
         const val EXTRA_SALE_INFORMATION: String = "EXTRA_SALE_INFORMATION"
     }
 
+    private val fireStore = FirebaseFirestore.getInstance()
```

Comment:
You should never use `FirebaseFirestore.getInstance()` but use `FirebaseProvider.getFirestore()` instead

Reply:
On top of that, the FirebaseAuth one line below is now also accessible through `FirebaseProvider.getAuth`

Although, now is probably a lie, it's still in a PR ^^ Well, I'll try to remember to update SaleInformationActivity if you merge before me.